src/app/container/info-tab/info-tab.component.html

```diff
@@ -267,14 +267,45 @@
           <!-- Public Topic -->
           <li *ngIf="isPublic">
             <span><strong matTooltip="Short description of the tool">Topic</strong>:&nbsp;</span>
-            <span>{{ tool?.topicSelection === TopicSelectionEnum.AUTOMATIC ? tool?.topicAutomatic : tool?.topicManual }}</span>
+            <span *ngIf="tool?.topicSelection === TopicSelectionEnum.AUTOMATIC">{{ tool?.topicAutomatic }}</span>
+            <span *ngIf="tool?.topicSelection === TopicSelectionEnum.MANUAL">{{ tool?.topicManual }}</span>
+            <span *ngIf="tool?.topicSelection === TopicSelectionEnum.AI">
+              {{ tool?.topicAI }}
+              <span
+                class="bubble ai-bubble"
+                matTooltip="This topic sentence was AI generated. For more information, please click the bubble to visit our docs"
```

Comment:
Not sure how I feel about saying "For more information, please click the bubble to visit our docs" since I don't think we explicitly say it else where. For example, on the Versions tab, the info icons link to documentation but the tooltips don't say anything about clicking it.

![image](https://github.com/dockstore/dockstore-ui2/assets/25287123/a012be10-dfb8-4d5b-aa1f-608759c07022)

Reply:
this could work too
I also noticed the circle ci comparison changed a little. 
@hyunnaye could you create the ticket for the docs?

![Screenshot from 2024-03-04 15-37-57](https://github.com/dockstore/dockstore-ui2/assets/1730679/1d9705e0-db87-4d6a-b4ca-4742c9ec6e55)

Reply:
> We should probably do something similar i.e. non-published workflows will not be sent for AI topic sentences @kathy-t (think it may already be the case, but double-checking)

That's correct, the topic generator uses the public TRS API to get descriptor content and we don't even provide a dockstore token to the topic generator

Reply:
Screenshots updated for the tooltip. And added a hover effect for the bubble

Reply:
Ticket created for the docs: https://ucsc-cgl.atlassian.net/browse/SEAB-6308